feature/credits-woocommerce (#1964)
mercadopago/devsite-docs: guides/woocommerce/mercado-credits.en.md

----[mla, mlm, mlb]----
# Installments without card

**Mercado Credits** is Mercado Pago’s financing method that allows paying in installments without having a card.

With this line of credit, administered by Mercado Pago, the payment is credited in full to the seller's account, while the customer can choose to pay in up to 12 fixed monthly installments, no card needed. The user just has to enter their Mercado Pago account (or create one), determine their available limit, and choose how many installments they want to pay in.
 

> **Mercado Credits** is currently offered in our Checkout Pro, and now it is also possible to access directly from the store checkout. To find out how to **display Mercado Crédito at the checkout of your store**, continue reading the steps below. To learn **how to set up Checkout Pro**, you can click [here](/developers/en/docs/woocommerce/payments-configuration/checkout-pro).

Reviewer: ```suggestion
**Mercado Credits** is currently offered in our Checkout Pro, and now it is also possible to access directly from the store checkout. To find out how to **display Mercado Crédito at the checkout of your store**, follow the steps below.
```

## Configure the payment method in your store's checkout

> WARNING
>
> Attention
>

> > In order to offer **Installments without a card** as a payment method in your WooCommerce store, you must first **update the Mercado Pago plugin** to the most recent version available. To do so, in your store’s admin panel go to **Plugins > Installed Plugins**, search for **Mercado Pago**, and click on **Update**.

Reviewer: ```suggestion
> In order to offer **Installments without a card** as a payment method in your WooCommerce store, you must first have Checkout Pro configured on your store. To learn how to **configure checkout**, click [here](/developers/en/docs/woocommerce/payments-configuration/checkout-pro). <br/></br>
> <br/></br>
> In addition, it is also necessary to **update the Mercado Pago plugin** to the latest available version. To do so, in your store’s admin panel go to **Plugins > Installed Plugins**, search for **Mercado Pago**, and click on **Update**.
```

With this line of credit, administered by Mercado Pago, the payment is credited in full to the seller's account, while the customer can choose to pay in up to 12 fixed monthly installments, no card needed. The user just has to enter their Mercado Pago account (or create one), determine their available limit, and choose how many installments they want to pay in.
 
**Mercado Credits** is currently offered in our Checkout Pro, and now it is also possible to access directly from the store checkout. To find out how to **display Mercado Crédito at the checkout of your store**, continue reading the steps below. To learn **how to set up Checkout Pro**, you can click [here](/developers/en/docs/woocommerce/payments-configuration/checkout-pro).

## Configure the payment method in your store's checkout

> WARNING
>
> Attention
>
> In order to offer **Installments without a card** as a payment method in your WooCommerce store, you must first **update the Mercado Pago plugin** to the most recent version available. To do so, in your store’s admin panel go to **Plugins > Installed Plugins**, search for **Mercado Pago**, and click on **Update**.

> 

Reviewer: ```suggestion
```
